src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -182,7 +182,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
    */
   loadFeedbackSession(): void {
     // load the course of the feedback session first
-    this.courseService.getCourse(this.courseId).subscribe((course: Course) => {
+    this.courseService.getCourseAsInstructor(this.courseId).subscribe((course: Course) => {
```

Comment:
Is this the only place where the API is used?

Reply:
Yup. IntelliJ only reports this when I searched for usages. The students endpoint is still utilizing STUDENT_COURSE and STUDENT_COURSES